todo_app/hypothesistest.py

```diff
@@ -0,0 +1,13 @@
+import pytest 
```

Comment:
This looks like it's probably for practise/preparation in M3; that's well worth doing but in a real project it would still be worth removing this again (even if you leave it in the Git history for future) in order to ensure the codebase doesn't include anything unnecessary that can distract